### PR TITLE
Filter index

### PR DESCRIPTION
**Pull request by Jacopobelli5**

### CODE REVIEW

**Review by ozdere — APPROVED** [review]
Looks good!

**Comment by sarhan-a** [comment]
Thanks Jacopo, do you think can shed some space by clustering min price and max price in a small dropdown?
Similar to this example:
Before
image
After
image

File: app/views/shared/_filter.html.erb [hunk 1]
```diff
@@ -1,16 +1,36 @@
...
+<%= f.label :max_price, class:"col-form-label" %>
+</div>
+<div class="col-auto">
+<%= f.search_field :price_lteq, class:"form-control" %>
+</div>
+
+<div class="form-group col-auto">
+<%= f.label :collection_date, class:"col-form-label" %>
+</div>
+<div class="col-auto">
```
Comment (sarhan-a) [inline app/views/shared/_filter.html.erb:32]: add pt-3 here to align with the filter button after it.

**Review by sarhan-a — CHANGES_REQUESTED** [review]

File: app/views/shared/_filter.html.erb (hunk 1 above)
Comment (Jacopobelli5) [reply app/views/shared/_filter.html.erb:32]: 👍🏻

**Review by sarhan-a — APPROVED** [review]

**Pull request merged**